### PR TITLE
Fix windows build

### PR DESCRIPTION
**Pull request by SylvainCorlay**

### CODE REVIEW

File: src/Makevars [hunk 1]
```diff
@@ -5,9 +5,9 @@
 # The full license is in the file LICENSE, distributed with this software.
 
 # Default make file
-PKG_CPPFLAGS=-std=c++17
+PKG_CPPFLAGS=-std=c++17 -I$(R_HOME)/../../Library/include
```
Comment (SylvainCorlay) [inline src/Makevars]: I can't think of a better way to address the Library prefix of conda, than by `$(R_HOME)/../../`.

**Comment by SylvainCorlay** [comment]
The windows failure at this point is due to mingw not supporting c++17. (We would need to expose more restricted headers to libmamba so that (too) modern C++ features are not required.)

Another issue we will face is that rhumba & R are built with mingw and libmamba with MSVC, which makes the C++ mangling different. We may need to expose a C API for what is required in rhumba.

File: src/Makevars (hunk 1 above)
Comment (marimeireles) [reply src/Makevars]: We're having this issue on Windows only right?
Or can you reproduce in a different OS?

**Comment by marimeireles** [comment]
>mingw not supporting c++17

I'm clueless on windows, but searching a bit the web [I found some solutions to use c++17](https://www.geeksforgeeks.org/complete-guide-to-install-c17-in-windows/) with mingw (I think).
Or is it just a hacky way to do it?

Do we have another solution? I saw that you pinged Serge at some point in the internal chat, but I can't find the message anymore. If you have any ideas on how to make this work I'm happy to look into it. Maybe I should do a dualboot with windows here.

**Comment by SylvainCorlay** [comment]
This is now included in #19, closing.

**Pull request closed without merging**